homeassistant/components/irm_kmi/config_flow.py

```diff
@@ -0,0 +1,147 @@
+
+            # Check if zone is in Benelux.
+            if not errors:
+                assert zone is not None  # Assert is here for mypy linting.
+                api_data = {}
+                try:
+                    async with asyncio.timeout(60):
+                        api_data = await IrmKmiApiClient(
+                            session=async_get_clientsession(self.hass),
+                            user_agent=USER_AGENT,
+                        ).get_forecasts_coord(
+                            {
+                                "lat": zone.attributes[ATTR_LATITUDE],
+                                "long": zone.attributes[ATTR_LONGITUDE],
+                            }
+                        )
+                except Exception:  # noqa: BLE001
+                    errors["base"] = "api_error"
+
+                if api_data.get("cityName", None) in OUT_OF_BENELUX:
+                    errors[CONF_ZONE] = "out_of_benelux"
+
+                if not errors:
+                    await self.async_set_unique_id(user_input[CONF_ZONE])
+                    self._abort_if_unique_id_configured()
+
+                    state = self.hass.states.get(user_input[CONF_ZONE])
+                    return self.async_create_entry(
+                        title=state.name if state else "IRM KMI",
+                        data={
+                            CONF_ZONE: user_input[CONF_ZONE],
+                            CONF_LANGUAGE_OVERRIDE: user_input[CONF_LANGUAGE_OVERRIDE],
+                        },
+                    )
+
+        return self.async_show_form(
+            step_id="user",
+            errors=errors,
+            description_placeholders={
+                "zone": user_input.get("zone", "") if user_input is not None else ""
+            },
+            data_schema=vol.Schema(
+                {
+                    vol.Required(CONF_ZONE): EntitySelector(
+                        EntitySelectorConfig(domain=ZONE_DOMAIN)
+                    ),
+                    vol.Required(
+                        CONF_LANGUAGE_OVERRIDE, default="none"
+                    ): SelectSelector(
+                        SelectSelectorConfig(
+                            options=CONF_LANGUAGE_OVERRIDE_OPTIONS,
+                            mode=SelectSelectorMode.DROPDOWN,
+                            translation_key=CONF_LANGUAGE_OVERRIDE,
+                        )
+                    ),
+                }
+            ),
+        )
+
+
+class IrmKmiOptionFlow(OptionsFlow):
+    """Option flow for the IRM KMI integration, help change the options once the integration was configured."""
+
+    def __init__(self, config_entry: IrmKmiConfigEntry) -> None:
+        """Initialize options flow."""
+        self.current_config_entry = config_entry
+
+    async def async_step_init(self, user_input: dict | None = None) -> ConfigFlowResult:
+        """Manage the options."""
+        if user_input is not None:
+            _LOGGER.debug(user_input)
```

Comment:
```suggestion
            _LOGGER.debug("Provided config user is: %s", user_input)
```

Reply:
Thanks for the suggested changes.  I implemented them in commit e7af3bbf30e8b3e1a0a4f62be5ff09af3347e9fb